services/wallet/activity/activity.go

```diff
@@ -675,6 +678,10 @@ func getActivityEntries(ctx context.Context, deps FilterDependencies, addresses
 				tokenOut = deps.tokenFromSymbol(&cID, tokenCode.String)
 			}
 
+			if activityType == SendAT && ZeroAddress == toAddress && tokenOut.TokenType == Native {
```

Comment:
Doesn't look that we're checking whether the ContractAddress field is non-zero here, this could be an ETH burn operation.

Reply:
Example: https://etherscan.io/tx/0xc12d48df1f4b2fb13106502819b77acff8cbc85e4f3cc948be4684e9808da30b